camera.py

```diff
@@ -0,0 +1,8 @@
+class Camera:
```

Comment:
Use more descriptive variable names. Instead of resolution and focal_length, consider using image_resolution and focal_length_mm